controllers/Category.js

```diff
@@ -0,0 +1,29 @@
+const User = require('../models/User')
+const Package = require('../models/Package')
+const Iternary = require('../models/Iternary')
+const Category = require('../models/Category')
+
+exports.getAllCategories = (req, res) => {
+  Category.find({}, 'name', function(err, categories) {
+    if (err) return err
+    res.send(categories)
+  })
+}
+
+exports.getOneCategory = (req, res) => {
+  Category.findById(req.params.categoryId)
+    .select('-__v')
```

Comment:
Do you have to do select('-__v') here? Since you have declared `versionKey: false`.